cardshowviewtakenpicturesview/src/main/java/br/com/stant/libraries/cardshowviewtakenpicturesview/camera/CameraPhotosAdapter.java

```diff
@@ -164,12 +195,23 @@ public void fileNotFound() {
 
             mCameraPhotosRecyclerViewBinding.setPhoto(cameraPhoto);
             mCameraPhotosRecyclerViewBinding.executePendingBindings();
-            mCameraPhotosRecyclerViewBinding.cardShowTakenPictureViewGeneralCircularImageView.setOnClickListener(
+            mCameraPhotosRecyclerViewBinding.cameraPhotoViewItemPhotoCircularImageView.setOnClickListener(
                     view -> mCameraFragment.showPreviewPicDialog(cameraPhoto)
             );
         }
 
+        @Override
+        public void onItemSelected() {
+            VibratorUtils.vibrate(mContext, 400);
```

Comment:
create a variable to describe number constant and the programmer not need see method declaration

Reply:
Created